### DISCUSSION TITLE
using GPU for unwrapping step?

### DISCUSSION THREAD

**Discussion by raviexplorer** [gmtsar/gmtsar#423] [General]
Can I also use gpu for the unwrapping step in GMTSAR ?
If yes then please tell how

**Comment by Xiaohua-Eric-Xu** [comment]
I am not sure what's a good gpu unwraper. Please let us know if you find one.

**Comment by chintals** [comment]
Unwrapping in GMTSAR is done by [SNAPHU](https://web.stanford.edu/group/radar/softwareandlinks/sw/snaphu/). And there are no known public versions of SNAPHU utilizing GPUs for unwrapping.  The c-based SNAPHU code needs to be rewritten in CUDA language.
